modules/configuring-egress-proxy.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * networking/aws_load_balancer_operator/configure-egress-proxy-aws-load-balancer-operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-configuring-egress-proxy_{context}"]
+= Configuring the AWS Load Balancer Operator to trust the Certificate Authority of the cluster wide egress proxy
+
+. Create the config map that contains the CA bundle in the `aws-load-balancer-operator` namespace by running the following command:
```

Comment:
We need to highlight somehow the fact that this configmap doesn't contain the CA bundle just now.
It will be after the second command is run. Maybe we can say something like:
```suggestion
. Create the config map to contain the CA bundle in the `aws-load-balancer-operator` namespace by running the following command:
```